### ISSUE TITLE
Safety checker crashes on DIV and IDIV

### ISSUE THREAD

**Issue by eponier** [jasmin-lang/jasmin#561]
The safety checker thinks that these instructions take 2 arguments, while they take 3. I do not know how to fix that properly.

**Comment by vbgl** [comment]
Can you please provide a minimal failing example? Thanks.

**Comment by eponier** [comment]
Sure, sorry!
```
export fn divEx1() -> reg u64 {
  reg u64 hi, lo, divisor, res;
  hi = 0;
  lo = 20;
  divisor = 5;
  ?{RAX=res} = #IDIV (hi, lo, divisor);
  return res;
}
```

Btw, I did not know that we could/had to do `?{RAX=res}`.

**Comment by eponier** [comment]
```
jasminc -checksafety div.jazz
```
```
Default checker parameters.

Analyzing function divEx1
Fatal error: exception File "src/utils.ml", line 43, characters 51-57: Assertion failed
```

**Comment by vbgl** [comment]
Thanks. Fixing the assertion failure should be easy, but making the correct safety checks seems trickier.

**Comment by vbgl** [comment]
I went for the easy fix in #574.
